src/web/app/actions/beambox/svgeditor-function-wrapper.ts

```diff
@@ -68,9 +69,20 @@ const funcs = {
         },
       ],
     });
-    if (file) svgEditor.handleFile(file);
+
+    if (file) {
+      if (file.type === 'image/svg+xml') {
+        webNeedConnectionWrapper(() => {
+          svgEditor.handleFile(file);
```

Comment:
There are other places calling `handleFile` such as drag & drop and pasting file in editor, so add the wrapper in `handleFile` SVG part may be better